src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -206,4 +208,43 @@ class BagsServletSpec extends TestSupportFixture
       body shouldBe s"Item $itemId not found"
     }
   }
+
+  it should "fail when done on an inactive/ hidden bag" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID") {
+      println(body)
+      status shouldBe 410
+      body.lines.mkString should include(s"Tried to retrieve an inactive bag: $bagID")
+    }
+  }
+
+  it should "fail when done on an item within an inactive/ hidden bag" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID/bag-info2.txt") {
+      status shouldBe 410
+      body.lines.mkString should include(s"Tried to retrieve an inactive bag: $bagID")
+    }
+  }
+
+  //TODO are these two cases below correct?
```

Comment:
I suppose this question if for @lindareijnhoudt : should an attempt to deactivate something (with or without Accept header) that is already inactive fail or silently succeed?